tests/cmc_facade_test.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import requests
+from unittest.mock import patch
+from facades.cmc_facade import get_btc_price
+
+@pytest.fixture
+def mock_btc_response():
+    return {
+        "data": {
+            "BTC": [{
+                "quote": {
+                    "USD": {
+                        "price": 45000.00
+                    }
+                }
+            }]
+        }
+    }
+
+def test_get_btc_price_success(mock_btc_response):
+    with patch('requests.get') as mock_get:
+        mock_get.return_value.json.return_value = mock_btc_response
```

Comment:
I think it's also important to test for status codes when developing callouts. 
`mock_get.return_value.status_code = 200` would be good.